lstchain/scripts/lstchain_find_pedestals.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""
+This script has to be launched (with no arguments) on a directory containing
+DL1 sub-run (i.e. not merged) LST real data files. It will find the
+event_id's of interleaved pedestals, and write them out into one hdf5 file for
+each sub-run, called pedestal_ids_RunXXXXX.YYYY.h5, which contains a single
+table with one column, 'event_id'
+"""
+
+import argparse
+import glob
+import os
+import numpy as np
+import pandas as pd
+
+from pathlib import Path
+from ctapipe.io import read_table
+from lstchain.paths import parse_dl1_filename
+
+parser = argparse.ArgumentParser(description="Interleaved Pedestal Finder")
+
+parser.add_argument(
+    '-o', '--output-dir', type=Path,
+    help='Path where to store the reco dl1 events',
+    default='./dl1_data/'
+)
+
+def main():
+
+    args = parser.parse_args()
+    output_dir = args.output_dir.absolute()
+    output_dir.mkdir(exist_ok=True)
+
+    files = glob.glob('dl1_LST-1.Run?????.????.h5')
+    files.sort()
+    if not files:
+        raise IOError("No input dl1 files found")
+    files.sort()
+
+    for file in files:
+        run_number = parse_dl1_filename(os.path.basename(file)).run
+        subrun_index = parse_dl1_filename(os.path.basename(file)).subrun
+
+        # Approximate interleaved pedestal frequency in Hz:
+        approximate_frequency = 50
+        if run_number > 2708:
+            approximate_frequency = 100
+
+        table = read_table(file, '/dl1/event/telescope/parameters/LST_LSTCam')
+        all_times = np.array(table['dragon_time'])
+        # probable interleaved flat-field events (just in case tag is
+        # faulty!):
```

Comment:
Why not rely on the event type for the FF events here? It should be tagged correctly by the source, or not?

Reply:
*should*,  yes...  It is the grammatical tense that makes me tense. I really prefer a straight test here.